weareinreach/inreach-catalog
# Refactor Disclaimer & Privacy Statement (Mobile)

- updates en-US.json : breaks up privacy text to account for external links. 
- refactors:
   -  privacy.js
   - disclaimer text.js
   - privacy mobile.js
   - language.js

- Asana ticket: Related to #117 

## PR Checklist




## How to Test
 - Launch the app (mobile view)
 - Click "More"
 - Click "Privacy Information"
 - Click "Privacy Statement and Disclaimer"
    - Verify that the Privacy Statement displays correctly
    - Verify that the Disclaimer Statement displays correctly
    - Verify that the "anonymised IP addresses" link works correctly
    - Verify that the opt out link (here) works correctly
